src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1975,54 +1975,79 @@ limitations under the License.
             </include>
             <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
```

Comment:
It's higher up, but why is this including Binding client?

Reply:
@chrisdecenzo should binding cluster be removed?

Reply:
As a commissioner, the tv-app needs to be able to create bindings and I think we needed this at one time so that tv-app would compile. Not sure if by including the controller we now have the binding client included

Reply:
Again, the _device type_ should not have Binding, because that is not in the spec.

If some specific implementation wants to have a binding client, it can do that.  Note that there is no binding client code of any sort if you include it; it's just a matter of what gets claimed via Descriptor.

Reply:
Removed.